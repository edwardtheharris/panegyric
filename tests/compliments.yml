- from: billybuck
  text: |
    You have very nice hair.