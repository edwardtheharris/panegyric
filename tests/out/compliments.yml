- {from: billybuck, send_date: '2021-07-26', text: "You have very nice hair\n"}
- {from: someguy, text: "You are so smart\n"}
- {from: someotherguy, text: My favorite}
