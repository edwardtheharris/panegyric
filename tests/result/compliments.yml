- from: billybuck
  text: You have very nice hair
  send_date: '2021-08-04'
- from: someguy
  text: You are so smart
- from: someotherguy
  text: My favorite
