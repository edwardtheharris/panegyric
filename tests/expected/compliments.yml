- from: billybuck
  text: |
    You have very nice hair
  send_date: '2021-07-26'
- from: someguy
  text: |
    You are so smart
- from: someotherguy
  text: |
    My favorite